scripts/python/api/tnode.py

```diff
@@ -84,7 +76,7 @@ def createV2Plugin(self, args=""):
     # Enable v2plugin on vagrant node
     def enableV2Plugin(self, args=""):
         ssh_object = self.sshConnect(self.username, self.password)
-        command = "docker plugin set " + os.environ.get("CONTIV_V2PLUGIN_NAME", "") + " iflist=eth2,eth3 plugin_name=" + os.environ["CONTIV_V2PLUGIN_NAME"] + args + " >> /tmp/netplugin.log 2>&1"
+        command = "docker plugin set " + os.environ.get("CONTIV_V2PLUGIN_NAME", "") + " ctrl_ip="+ self.addr + " control_url=" + self.addr + ":9999 iflist=eth2,eth3 plugin_name=" + os.environ["CONTIV_V2PLUGIN_NAME"] + args + " >> /tmp/netplugin.log 2>&1"
```

Comment:
For my understanding, does this take arguments for both netplugin and netmaster and start both of them? If so, will the logs of both netmaster and netplugin be redirected to /tmp/netplugin.log?